Halide versions of *scal, *copy BLAS benchmarks segfault

```shell
mkdir build
cd build
cmake .. -DWITH_TESTS=OFF
make -j12
make l1_benchmarks -k
```

Output:
```shell
[  0%] Built target build_halide_h
...
[100%] Built target halide_benchmarks
  Halide              sasum      16            0.027750             1.15315
[100%] Built target halide_l1_benchmark_sasum_16
[100%] Built target l1_benchmark_sasum_16
  Halide              sasum    2080            0.136742             30.4223
[100%] Built target halide_l1_benchmark_sasum_2080
[100%] Built target l1_benchmark_sasum_2080
  Halide              sasum     288            0.038761             14.8602
[100%] Built target halide_l1_benchmark_sasum_288
[100%] Built target l1_benchmark_sasum_288
  Halide              sasum    1056            0.075958             27.8047
[100%] Built target halide_l1_benchmark_sasum_1056
[100%] Built target l1_benchmark_sasum_1056
  Halide              sasum      64            0.031798             4.02536
[100%] Built target halide_l1_benchmark_sasum_64
[100%] Built target l1_benchmark_sasum_64
[100%] Built target l1_benchmark_sasum
  Halide              dasum    2080            0.242287             17.1697
[100%] Built target halide_l1_benchmark_dasum_2080
[100%] Built target l1_benchmark_dasum_2080
  Halide              dasum      64            0.033807             3.78622
[100%] Built target halide_l1_benchmark_dasum_64
[100%] Built target l1_benchmark_dasum_64
  Halide              dasum     288            0.050051             11.5083
[100%] Built target halide_l1_benchmark_dasum_288
[100%] Built target l1_benchmark_dasum_288
  Halide              dasum      16            0.027946             1.14509
[100%] Built target halide_l1_benchmark_dasum_16
[100%] Built target l1_benchmark_dasum_16
  Halide              dasum    1056            0.129845             16.2656
[100%] Built target halide_l1_benchmark_dasum_1056
[100%] Built target l1_benchmark_dasum_1056
[100%] Built target l1_benchmark_dasum
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_288.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_288' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_288] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_288.dir/build' not remade because of errors.
CMakeFiles/Makefile2:16520: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_288.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_288.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_1056.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_1056' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_1056] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_1056.dir/build' not remade because of errors.
CMakeFiles/Makefile2:16060: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_1056.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_1056.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_2080.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_2080' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_2080] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_2080.dir/build' not remade because of errors.
CMakeFiles/Makefile2:14277: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_2080.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_2080.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_64.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_64' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_64] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_64.dir/build' not remade because of errors.
CMakeFiles/Makefile2:14633: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_64.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_64.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_16.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_16' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_16] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_16.dir/build' not remade because of errors.
CMakeFiles/Makefile2:15314: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_16.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dscal_16.dir/all] Error 2
  Halide              ddot      64            0.040874             3.13154
[100%] Built target halide_l1_benchmark_ddot_64
[100%] Built target l1_benchmark_ddot_64
  Halide              ddot    2080            0.767700             5.41878
[100%] Built target halide_l1_benchmark_ddot_2080
[100%] Built target l1_benchmark_ddot_2080
  Halide              ddot      16            0.029182             1.09655
[100%] Built target halide_l1_benchmark_ddot_16
[100%] Built target l1_benchmark_ddot_16
  Halide              ddot     288            0.120422             4.78316
[100%] Built target halide_l1_benchmark_ddot_288
[100%] Built target l1_benchmark_ddot_288
  Halide              ddot    1056            0.385330             5.48101
[100%] Built target halide_l1_benchmark_ddot_1056
[100%] Built target l1_benchmark_ddot_1056
[100%] Built target l1_benchmark_ddot
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_2080.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_2080' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_2080] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_2080.dir/build' not remade because of errors.
CMakeFiles/Makefile2:12952: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_2080.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_2080.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_64.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_64' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_64] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_64.dir/build' not remade because of errors.
CMakeFiles/Makefile2:15122: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_64.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_64.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_16.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_16' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_16] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_16.dir/build' not remade because of errors.
CMakeFiles/Makefile2:20133: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_16.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_16.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_288.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_288' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_288] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_288.dir/build' not remade because of errors.
CMakeFiles/Makefile2:18665: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_288.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_288.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_1056.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_1056' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_1056] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_1056.dir/build' not remade because of errors.
CMakeFiles/Makefile2:14835: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_1056.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_scopy_1056.dir/all] Error 2
  Halide              daxpy      64            0.034555             3.70419
[100%] Built target halide_l1_benchmark_daxpy_64
[100%] Built target l1_benchmark_daxpy_64
  Halide              daxpy    1056            0.178665              11.821
[100%] Built target halide_l1_benchmark_daxpy_1056
[100%] Built target l1_benchmark_daxpy_1056
  Halide              daxpy     288            0.055523             10.3741
[100%] Built target halide_l1_benchmark_daxpy_288
[100%] Built target l1_benchmark_daxpy_288
  Halide              daxpy      16            0.029023             1.10257
[100%] Built target halide_l1_benchmark_daxpy_16
[100%] Built target l1_benchmark_daxpy_16
  Halide              daxpy    2080            0.372070             11.1807
[100%] Built target halide_l1_benchmark_daxpy_2080
[100%] Built target l1_benchmark_daxpy_2080
[100%] Built target l1_benchmark_daxpy
  Halide              sdot     288            0.068329             8.42982
[100%] Built target halide_l1_benchmark_sdot_288
[100%] Built target l1_benchmark_sdot_288
  Halide              sdot    2080            0.393539             10.5707
[100%] Built target halide_l1_benchmark_sdot_2080
[100%] Built target l1_benchmark_sdot_2080
  Halide              sdot    1056            0.208130             10.1475
[100%] Built target halide_l1_benchmark_sdot_1056
[100%] Built target l1_benchmark_sdot_1056
  Halide              sdot      64            0.032736             3.91009
[100%] Built target halide_l1_benchmark_sdot_64
[100%] Built target l1_benchmark_sdot_64
  Halide              sdot      16            0.028867             1.10854
[100%] Built target halide_l1_benchmark_sdot_16
[100%] Built target l1_benchmark_sdot_16
[100%] Built target l1_benchmark_sdot
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_2080.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_2080' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_2080] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_2080.dir/build' not remade because of errors.
CMakeFiles/Makefile2:13048: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_2080.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_2080.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_16.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_16' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_16] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_16.dir/build' not remade because of errors.
CMakeFiles/Makefile2:15896: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_16.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_16.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_64.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_64' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_64] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_64.dir/build' not remade because of errors.
CMakeFiles/Makefile2:14601: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_64.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_64.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_1056.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_1056' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_1056] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_1056.dir/build' not remade because of errors.
CMakeFiles/Makefile2:16260: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_1056.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_1056.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_288.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_288' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_288] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_288.dir/build' not remade because of errors.
CMakeFiles/Makefile2:19743: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_288.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_sscal_288.dir/all] Error 2
  Halide              saxpy    2080            0.171951             24.1929
[100%] Built target halide_l1_benchmark_saxpy_2080
[100%] Built target l1_benchmark_saxpy_2080
  Halide              saxpy    1056            0.096100             21.9771
[100%] Built target halide_l1_benchmark_saxpy_1056
[100%] Built target l1_benchmark_saxpy_1056
  Halide              saxpy     288            0.042435             13.5738
[100%] Built target halide_l1_benchmark_saxpy_288
[100%] Built target l1_benchmark_saxpy_288
  Halide              saxpy      16            0.026495             1.20778
[100%] Built target halide_l1_benchmark_saxpy_16
[100%] Built target l1_benchmark_saxpy_16
  Halide              saxpy      64            0.031528             4.05982
[100%] Built target halide_l1_benchmark_saxpy_64
[100%] Built target l1_benchmark_saxpy_64
[100%] Built target l1_benchmark_saxpy
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_1056.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_1056' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_1056] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_1056.dir/build' not remade because of errors.
CMakeFiles/Makefile2:11720: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_1056.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_1056.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_64.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_64' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_64] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_64.dir/build' not remade because of errors.
CMakeFiles/Makefile2:14109: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_64.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_64.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_2080.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_2080' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_2080] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_2080.dir/build' not remade because of errors.
CMakeFiles/Makefile2:12686: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_2080.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_2080.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_16.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_16' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_16] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_16.dir/build' not remade because of errors.
CMakeFiles/Makefile2:14537: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_16.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_16.dir/all] Error 2
Segmentation fault (core dumped)
apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_288.dir/build.make:57: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_288' failed
make[3]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_288] Error 139
make[3]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_288.dir/build' not remade because of errors.
CMakeFiles/Makefile2:13847: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_288.dir/all' failed
make[2]: *** [apps/linear_algebra/benchmarks/CMakeFiles/halide_l1_benchmark_dcopy_288.dir/all] Error 2
make[2]: Target 'apps/linear_algebra/benchmarks/CMakeFiles/l1_benchmarks.dir/all' not remade because of errors.
CMakeFiles/Makefile2:15027: recipe for target 'apps/linear_algebra/benchmarks/CMakeFiles/l1_benchmarks.dir/rule' failed
make[1]: *** [apps/linear_algebra/benchmarks/CMakeFiles/l1_benchmarks.dir/rule] Error 2
make[1]: Target 'l1_benchmarks' not remade because of errors.
Makefile:5605: recipe for target 'l1_benchmarks' failed
make: *** [l1_benchmarks] Error 2
```

--- a/apps/linear_algebra/src/CMakeLists.txt
+++ b/apps/linear_algebra/src/CMakeLists.txt
@@ -21,12 +21,12 @@ halide_generator(dgemm.generator SRCS blas_l3_generators.cpp)
 function(add_halide_blas_library)
   set(options )
   set(oneValueArgs TARGET NAME)
-  set(multiValueArgs GENERATOR_ARGS)
+  set(multiValueArgs)
   cmake_parse_arguments(args "" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
   halide_library_from_generator(${args_TARGET}
       GENERATOR ${args_NAME}.generator
       HALIDE_TARGET_FEATURES no_asserts no_bounds_query
-      GENERATOR_ARGS ${args_ARGN}
+      ${args_UNPARSED_ARGUMENTS}
   )
   target_link_libraries(halide_blas PUBLIC ${args_TARGET})
 endfunction()

